src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -81,6 +81,66 @@ public void canPutAndGet() {
     assertThat(getRulesText(), is(rule));
   }
 
+  @Test
+  public void canUpdateCirculationRuleInDifferentPolicyOrder() {
+
+    UUID loanPolicy = UUID.randomUUID();
+    UUID requestPolicy = UUID.randomUUID();
+    UUID noticePolicy = UUID.randomUUID();
+    UUID overdueFinePolicy = UUID.randomUUID();
+    UUID lostItemFeePolicy = UUID.randomUUID();
+
+    loanPoliciesFixture.create(new LoanPolicyBuilder()
+      .withId(loanPolicy)
+      .withName("Example LoanPolicy " + loanPolicy));
+    noticePoliciesFixture.create(new NoticePolicyBuilder()
+      .withId(noticePolicy)
+      .withName("Example NoticePolicy " + noticePolicy));
+    requestPoliciesFixture.allowAllRequestPolicy(requestPolicy);
+    overdueFinePoliciesFixture.create(new OverdueFinePolicyBuilder()
+      .withId(overdueFinePolicy)
+      .withName("Example OverdueFinePolicy " + overdueFinePolicy));
+    lostItemFeePoliciesFixture.create(new LostItemFeePolicyBuilder()
+      .withId(lostItemFeePolicy)
+      .withName("Example lostItemPolicy " + lostItemFeePolicy));
+
+    String ruleInDiffPolicyOrder1 = "priority: t, s, c, b, a, m, g\n" +
+      "fallback-policy: i %s o %s n %s r %s l %s \n";
+    String ruleInDiffPolicyOrder2 = "priority: t, s, c, b, a, m, g\n" +
+      "fallback-policy: o %s i %s r %s n %s l %s \n";
+    String ruleInDiffPolicyOrder3 = "priority: t, s, c, b, a, m, g\n" +
+      "fallback-policy: o %s i %s l %s n %s r %s \n";
+    String ruleInDiffPolicyOrder4 = "priority: t, s, c, b, a, m, g\n" +
+      "fallback-policy: r %s l %s i %s o %s n %s \n";
+    String ruleInDiffPolicyOrder5 = "priority: t, s, c, b, a, m, g\n" +
+      "fallback-policy: o %s i %s l %s n %s r %s \n";
```

Comment:
This checks the order in the fallback-policy.
Can you also add two tests for the order in a regular rule?

Reply:
added